ygot/render.go

```diff
@@ -298,9 +298,9 @@ func (g *gnmiPath) StripPrefix(pfx *gnmiPath) (*gnmiPath, error) {
 	return newPathElemGNMIPath(g.pathElemPath[len(pfx.pathElemPath):]), nil
 }
 
-// GNMINotificationsConfig specifies arguments determining how the
+// GNMIConfig specifies arguments determining how the
 // gNMI output should be created by ygot.
-type GNMINotificationsConfig struct {
+type GNMIConfig struct {
```

Comment:
I would create an alternate type here -- since the 'string slice' path is something that is to be deprecated in gNMI, so for the `Set` case, where we're just implementing this API I think we do not need to support this.